clojure_liberator/src/clojure_liberator/data_access.clj

```diff
@@ -0,0 +1,22 @@
+(ns clojure-liberator.data-access)
+(use 'korma.db)
+(use 'korma.core)
+(require '[clojure.string :as str])
+
+(defdb eboshi (mysql {:db "eboshi_test"
+                       :user "root"
+                       :password ""
+                       ;; optional keys
+                       }))
+
+(declare clients users)
+
+(defentity clients)
+
+(defentity users)
```

Comment:
Cool, just curious. Out of all the languages in this repo, Clojure seems most similar to Haskell, so I'm curious about the differences. In Haskell, the compiler often demands you to declare all the things. ;)